test/api.scan.js

```diff
@@ -135,6 +135,17 @@ describe('picomatch', () => {
       assert.equal(base('path/foo/'), 'path/foo/');
       assert.equal(base('path/foo/bar.js'), 'path/foo/bar.js');
     });
+
+    it('should not return glob when noext is true', () => {
+      assert.deepEqual(scan('./foo/bar/*.js', {noext: true }), {
```

Comment:
Seems there is no rule in eslint for checking the spacing. Should I raise a PR separately for this?

Reply:
do you mean this spacing: `{noext`? Either way, that would be great. thanks!